packages/material-ui-unstyled/src/SliderUnstyled/SliderUnstyled.d.ts

```diff
@@ -26,6 +26,51 @@ export interface SliderTypeMap<P = {}, D extends React.ElementType = 'span'> {
      * A string value that provides a user-friendly name for the current value of the slider.
      */
     'aria-valuetext'?: string;
+    /**
+     * Override or extend the styles applied to the component.
+     */
+    classes?: {
```

Comment:
Should this change be moved into a different pull request?

Reply:
Yep, done